src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -199,8 +202,16 @@ private String validateSortClausePath(String path) {
         throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
             "path must be represented as a non-empty string");
       }
-      throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
-          "path ('%s') cannot start with `$`", path);
+      // But allow "well-known" fields
+      switch (path) {
+        case DocumentConstants.Fields.LEXICAL_CONTENT_FIELD,
+            DocumentConstants.Fields.VECTOR_EMBEDDING_FIELD,
+            DocumentConstants.Fields.VECTOR_EMBEDDING_TEXT_FIELD -> {}
+        default ->
+            throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
+                "path ('%s') cannot start with '$' (except for pseudo-fields '$lexical', '$vector' and '$vectorize')",
```

Comment:
nit: if I was to nit pick, when read in  full from the tests: 

> Invalid sort clause path: path ('$gt') cannot start with '$'

this can be read at the problem is `gt` should not start with `$`. 
Is there a way to tweak the message

Reply:
It is, but not sure how -- field paths cannot start with dollar sign (except for system ones). Suggestions?